cinema/views.py

```diff
@@ -0,0 +1,43 @@
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+
+from cinema.models import Movie
+from cinema.serializers import MovieSerializer
+
+
+@api_view(["GET", "POST"])
+def movie_list(request):
+    if request.method == "GET":
+        movies = Movie.objects.all()
+        serializer = MovieSerializer(movies, many=True)
+        return Response(serializer.data, status=status.HTTP_200_OK)
+
+    if request.method == "POST":
+        serializer = MovieSerializer(data=request.data)
+
+        if serializer.is_valid(raise_exception=True):
+            serializer.save()
+            return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+        return Response(serializer.errors, status=status.HTTP_400_BAD_REQUEST)

```

Comment:
invalid approach.
do like u did below
![image](https://github.com/mate-academy/py-movie-api/assets/86322722/44684758-a8e5-4e58-b976-e110dd5b61f3)

Reply:
@alim2709 
your PUT realization was correct!
change this code block like PUT realization

Reply:
change this code block with validation like in others cases